bin/i18n/resources/dashboard/course_content/sync_in.rb

```diff
@@ -16,26 +16,19 @@ module CourseContent
         class SyncIn < I18n::Utils::SyncInBase
           def process
             prepare_level_content
-            progress_bar.progress = 25
+            progress_bar.progress = 50
 
             prepare_project_content
-            progress_bar.progress = 50
+            progress_bar.progress = 75
 
             write_to_yml(VARIABLE_NAMES_TYPE, variable_strings)
-            progress_bar.progress = 65
+            progress_bar.progress = 90
 
             write_to_yml(PARAMETER_NAMES_TYPE, parameter_strings)
-            progress_bar.progress = 80
-
-            redact_level_content
           end
 
           private
 
-          def valid_source_file?(source_file_path)
```

Comment:
I'm guessing you removed this method because now it is only used in one place. However, I think it could be helpful to keep the method name  because it can make the code a little bit more readable. It would also be nice if there the was some documentation in the code about why we are skipping a script.

Reply:
I'm afraid that on the contrary, it will make the code a little bit more difficult to understand because we need to look up to check what "valid source file" means. But I'm not against this idea.